### PR TITLE
Update dotnet-watch.md

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnetcore/tutorials/dotnet-watch.md [hunk 1]
```diff
@@ -210,3 +210,14 @@ Some configuration options can be passed to `dotnet watch` through environment v
...
 | `DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER`   | `dotnet watch run` attempts to launch browsers for web apps with `launchBrowser` configured in *launchSettings.json*. If set to "1" or "true", this behavior is suppressed. |
 | `DOTNET_WATCH_SUPPRESS_BROWSER_REFRESH`   | `dotnet watch run` attempts to refresh browsers when it detects file changes. If set to "1" or "true", this behavior is suppressed. This behavior is also suppressed if `DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER` is set. |
+
+## Browser refresh
+
+`dotnet watch` injects a scripts in to the app to allow it to refresh the browser when content needs to be updated. In some scenarios, such as when the app enables response compression, `dotnet watch` might be unable to inject the script. In such a case, you may manually inject the script in to your application. For example, to configure your webapp to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```
Comment (danroth27) [inline aspnetcore/tutorials/dotnet-watch.md:216]:
```suggestion
`dotnet watch` injects a scripts in to the app to allow it to refresh the browser when content needs to be updated. In some scenarios, such as when the app enables response compression, `dotnet watch` might be unable to inject the script. In such a case, you may manually inject the script in to your application when in development. For example, to configure your webapp to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```

Comment (Rick-Anderson) [reply aspnetcore/tutorials/dotnet-watch.md:216]: Dan's suggestion plus mine, all in one

```suggestion
`dotnet watch` injects a script in to the app that allows it to refresh the browser when the content changes. In some scenarios, such as when the app enables response compression, `dotnet watch` might be unable to inject the script. In such a case, manually inject the script in to the app when in development. For example, to configure the  web app to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```

File: aspnetcore/tutorials/dotnet-watch.md [hunk 2]
```diff
@@ -210,3 +210,14 @@ Some configuration options can be passed to `dotnet watch` through environment v
...
 | `DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER`   | `dotnet watch run` attempts to launch browsers for web apps with `launchBrowser` configured in *launchSettings.json*. If set to "1" or "true", this behavior is suppressed. |
 | `DOTNET_WATCH_SUPPRESS_BROWSER_REFRESH`   | `dotnet watch run` attempts to refresh browsers when it detects file changes. If set to "1" or "true", this behavior is suppressed. This behavior is also suppressed if `DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER` is set. |
+
+## Browser refresh
+
+`dotnet watch` injects a scripts in to the app to allow it to refresh the browser when content needs to be updated. In some scenarios, such as when the app enables response compression, `dotnet watch` might be unable to inject the script. In such a case, you may manually inject the script in to your application when in development. For example, to configure your webapp to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```
Comment (pranavkm) [inline aspnetcore/tutorials/dotnet-watch.md:216]:
```suggestion
`dotnet watch` injects a script in to the app that allows it to refresh the browser when the content changes. In some scenarios, such as when the app enables response compression, `dotnet watch` might be unable to inject the script. For such cases in development, manually inject the script into the app. For example, to configure the  web app to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```

**Comment by pranavkm** [comment]
@Rick-Anderson good to go

File: aspnetcore/tutorials/dotnet-watch.md [hunk 3]
```diff
@@ -210,3 +210,14 @@ Some configuration options can be passed to `dotnet watch` through environment v
...
 | `DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER`   | `dotnet watch run` attempts to launch browsers for web apps with `launchBrowser` configured in *launchSettings.json*. If set to "1" or "true", this behavior is suppressed. |
 | `DOTNET_WATCH_SUPPRESS_BROWSER_REFRESH`   | `dotnet watch run` attempts to refresh browsers when it detects file changes. If set to "1" or "true", this behavior is suppressed. This behavior is also suppressed if `DOTNET_WATCH_SUPPRESS_LAUNCH_BROWSER` is set. |
+
+## Browser refresh
+
+`dotnet watch` injects a script in to the app that allows it to refresh the browser when the content changes. In some scenarios, such as when the app enables response compression, `dotnet watch` might be unable to inject the script. For such cases in development, manually inject the script into the app. For example, to configure the  web app to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/dotnet-watch.md:216]:
```suggestion
`dotnet watch` injects a script into the app that allows it to refresh the browser when the content changes. In some scenarios, such as when the app enables response compression, `dotnet watch` might ***not*** be unable to inject the script. For such cases in development, manually inject the script into the app. For example, to configure the  web app to manually inject the script, update the layout file to include `_framework/aspnet-browser-refresh.js`:
```

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**